### DISCUSSION TITLE
Using DKAN and Drupal 9

### DISCUSSION THREAD

**Discussion by Reptilo17** [GetDKAN/dkan#3982] [Q&A]
Hi,
I have installed D9 and Dkan last version. I can add datasets but the content of the node appears in Json format. I don't understand how to have a user friendly content type as in the D7 version and the documentation found doesn't help me. Thanks. Xavier

**Comment by janette** [comment]
@Reptilo17 if you go to the node/ID route, you should see your dataset information in a readable format similar to the D7 version. If you do not want to use the react frontend you will need to uninstall the DKAN JS Frontend module and customize the routing to use the node ID routes.

**Comment by Reptilo17** [comment]
Thanks but when I go for example to node/7 I have the Json format.
JSON Metadata
{"@type":"dcat:Dataset","title":"Test 2","identifier":"c949d926-97ff-4f51-87ba-8b37c477ea18","description":"Desc 2","accessLevel":"public","accrualPeriodicity":"R\/P10Y","modified":"2023-06-25T22:00:00+00:00","publisher":{"@type":"org:Organization","name":"Moi"},"contactPoint":{"fn":"moi","hasEmail":"mail"},"keyword":["Tag"]}
Type
dataset

**Comment by Reptilo17** [comment]
Hello,
I don't understand how to view the node as in the D7 version. When I edit the node I have all the fields as usual but when I save it, it  appears in a Json format.

**Comment by janette** [comment]
@Reptilo17 Newer versions of DKAN now include a twig template that will display the metadata in a more usable way if you go to the node view. You can also add your own custom template as well. Adding the data preview grid is a feature that is in our roadmap but has no definitive date. Have you tried using the react frontend?

**Comment by Reptilo17** [comment]
Thanks for your response.
I am using DKAN2 with Drupal 9.5.9.
I have uninstalled the DKAN JS Frontend but have always Json format as nodes.
It seems more difficult to have a plug and play Open Data platform using Dkan2

**Comment by Reptilo17** [comment]
I'm sorry but I really don't understand the way to have a node without the JSON format. I have desinstalled the DKAN JS Frontend and Front End modules but when I go to node/ID I have Title and JSON but don't have all the fields like in D7 DKAN.

**Reply by janette** [reply]
@Reptilo17 if you go to the Drupal route for the node do you see something like this: https://demo.getdkan.org/node/8
